launch.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+extra=""
+if [ -n "$JMX_PORT" ] ; then
+  extra="\
+  -Djava.rmi.server.hostname=localhost \
+  -Dcom.sun.management.jmxremote \
+  -Dcom.sun.management.jmxremote.port=$JMX_PORT \
```

Comment:
Can we pass all extra parameters as env-file?

Reply:
Done